server/game/cards/01_SOR/units/BodhiRook.ts

```diff
@@ -0,0 +1,33 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class BodhiRook extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '7257556541',
+            internalName: 'bodhi-rook#imperial-defector'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'Look at an opponent\'s hand and discard a non-unit card from it.',
+            immediateEffect: AbilityHelper.immediateEffects.sequential([
+                AbilityHelper.immediateEffects.lookAt((context) => ({
+                    target: context.player.opponent.hand.sort((a, b) => a.title.localeCompare(b.title)),
+                    sendChatMessage: true
+                })),
+                AbilityHelper.immediateEffects.discardCardsFromHand((context) => ({
+                    cardTypeFilter: WildcardCardType.Any,
+                    cardCondition: (card) => !card.isUnit(),
+                    discardingPlayerType: RelativePlayer.Opponent,
```

Comment:
Can we create a `discardCardsFromOpponentHand()` helper which hides this parameter? I think it is needed but it's probably confusing to understand how to set it correctly

Reply:
Yes created this helper function -- much easier to use